fdbclient/ManagementAPI.actor.cpp

```diff
@@ -3021,6 +3021,112 @@ ACTOR Future<Void> finalizeBulkLoadTask(Database cx, KeyRange range, UID taskId)
 	return Void();
 }
 
+// This is the only place to update job history map. So, we check the number of job history entries here is suffient
+// to maintain the number of job in the history is no more than BULKLOAD_JOB_HISTORY_COUNT_MAX.
```

Comment:
nit: that the number of jobs in the history...

Reply:
changed